tests/conftest.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime
+
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session, sessionmaker
+
+from app.database.models import Event, Invitation, User, Base
+
+
+@pytest.fixture(scope="session")
+def engine():
+    print("TestCase: Using sqlite database")
+    return create_engine('sqlite:///', echo=False)
+
+
+@pytest.fixture(scope="session")
+def session(engine):
+    sessionmaker_ = sessionmaker(bind=engine)
+    session = sessionmaker_()
+    Base.metadata.create_all(engine)
+
+    yield session
+
+    session.close()
+
+
+@pytest.fixture
+def user(session: Session) -> User:
+    test_user = create_model(
+        session, User,
+        username='test_username',
+        password='test_password',
+        email='test.email@gmail.com',
+    )
+    yield test_user
+    delete_instance(session, test_user)
```

Comment:
I like that you thought of the concept of DRY :)
Personally, I prefer to read tests with their logic as clear as possible - it's easier to understand what is going on in the test, and when.
So, I'm not sure if `delete_instance` function is more useful or more helps tests to be ambiguous :)
What do you think?